generator/config/stage/search.yaml

```
@@ -3,16 +3,92 @@ name: $search
 link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/search/'
 type:
     - stage
-encode: single
+encode: search
 description: |
     Performs a full-text search of the field or fields in an Atlas collection.
     NOTE: $search is only available for MongoDB Atlas clusters, and is not available for self-managed deployments.
 arguments:
     -
-        name: search
+        name: operator
+        type:
+            - searchOperator
+        description: |
+            Operator to search with.  You can provide a specific operator or use
+            the compound operator to run a compound query with multiple operators.
+    -
+        name: index
+        optional: true
+        type:
+            - string
+        description: |
+            Name of the Atlas Search index to use. If omitted, defaults to default.
```

```suggestion
            Name of the Atlas Search index to use. If omitted, defaults to "default".
```

Would be helpful to clarify the string literal here.